code/game/objects/structures/tables_racks.dm

```diff
@@ -585,22 +585,15 @@ CREATION_TEST_IGNORE_SUBTYPES(/obj/structure/table)
 	smoothing_flags = NONE
 	smoothing_groups = null
 	canSmoothWith = null
-	can_buckle = 1
-	buckle_lying = NO_BUCKLE_LYING
-	buckle_requires_restraints = 1
+	can_buckle = TRUE
+	buckle_lying = 90
 	var/mob/living/carbon/human/patient = null
 	var/obj/machinery/computer/operating/computer = null
 
 /obj/structure/table/optable/Initialize(mapload)
 	. = ..()
 	initial_link()
-
-/obj/structure/table/optable/ComponentInitialize()
-	. = ..()
-	var/static/list/loc_connections = list(
-		COMSIG_ATOM_ENTERED = PROC_REF(table_entered),
-	)
-	AddElement(/datum/element/connect_loc, loc_connections)
+	RemoveElement(/datum/element/climbable)
```

Comment:
I would just put a var on the parent, and a if statement for that var on initialize.

Better than add element parent remove element child snowflake stuff